Update EIP-1: Reference ERCs by ERC-X (#5273)
ethereum/EIPs: EIPS/eip-1.md

### EIP numbers

> When referring to an EIP by number, it must be written in the hyphenated form `EIP-X` where `X` is the EIP's assigned number. If the referenced EIP has a `category` of `ERC`, then it must be written in the hyphenated form `ERC-X` instead, where `X` is the EIP's assigned number.

Author: Is what I have now better?

> When referring to an EIP by number, if it does not have a `category` of `ERC`, it must be written in the hyphenated form `EIP-X` where `X` is the EIP's assigned number. If the referenced EIP has a `category` of `ERC`, then it must be written in the hyphenated form `ERC-X`, where `X` is the EIP's assigned number.

Reviewer: Because I _love_ bikeshedding so much, I think it'd be better to remove the negation:

```suggestion
When referring to an EIP with a `category` of `ERC`, it must be written in the hyphenated form `ERC-X` where `X` is that EIP's assigned number. When referring to EIPs with any other `category`, it must be written in the hyphenated form `EIP-X` where `X` is that EIP's assigned number.
```
